tests/content/entity/zone/create/create.js

```diff
@@ -1,31 +1,139 @@
 // Enabled draw zone bounding box and stack to visualize the stack of zone components
```

Comment:
I would suggest to keep the create test case the way it is, and add this codein the hirearchy test case